package.json

```diff
@@ -30,9 +30,9 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "@fastify/jwt": "^5.0.0",
```

Comment:
This includes the v4:
https://github.com/fastify/fastify-jwt/releases/tag/v4.0.0

We should keep the v3 till fastify v4

Reply:
Cheers @Eomm, will close this for now and revisit with v4's release

Reply:
@Eomm why?